.env.sample

```diff
@@ -1,9 +1,7 @@
 # Note: If you are using Azure App Service, go to your app's Configuration,
 # and then set the following values into your app's "Application settings".
-
-FLASK_DEBUG=True
+CLIENT_ID=<client id>
+CLIENT_SECRET=<client secret>
 # Expects a full tenant name such as "contoso.onmicrosoft.com", or its GUID
 # Or leave it undefined if you are building a multi-tenant app
 #TENANT_ID=<tenant id>
```

Comment:
Reordered to put optional one last, and also so that the comment is clearly scoped to it.

Reply:
I love your scope mindset.

For what it's worth, that is also why I prefer using inline comment in my Python source code. Inline comment always stays with the line it comments on. That benefit is well worth the price of my lines occasionally run longer than 80-ish letters. (And that is also why I reconcile with myself by thinking "inline comments does not count into line length". :wink: )

Reply:
Yep that's true. Hopefully IDEs and editors keep improving their ability to handle long lines.